app/assets/stylesheets/_variables.scss

```diff
@@ -170,16 +210,18 @@ $font_color_paginate_link: $colors_lightgrey;
 $bg_color_paginate_link_hover: $colors_greybrown;
 $font_color_page_visited: $colors_brown;
 $bg_color_page_current: $colors_green;
-$font_color_page_disabled: $colors_black;
+$font_color_page_disabled: $font_color_page;
 $bg_color_post_icon: $colors_greygreen;
```

Comment:
Could we rename these pagination variables to use `paginate` not `page`? I've been thinking of `page` as mostly the background to the overall website page, not the pagination page links, and didn't notice the ambiguity when reviewing the other documents.

I think this should then stay `$colors_black`, or a new `$font_color_paginate` should be made which this refers to, and that refers to `$colors_black`?